.gitignore

```diff
@@ -32,3 +32,5 @@ node_modules
 .vscode
 
 dist
+
+!tests/generate-data/dist
```

Comment:
Wh ydo we have to put `foo.js` and `bar.js` inside `dist` folder? 
Since we regenerating why not change path to more appropriate. Let's put them to `src` folder and let browserify output to `dist`

Reply:
👍 I was trying to minimize the changes to the test files and results, but moving them to source makes a better example.

Reply:
I guess we don't need this line anymore and can delete `&& rimraf dist/foo.js` in package,json